### ISSUE TITLE
Individual Character Coordinates 

### ISSUE THREAD

**Issue by gdabhishek** [Calamari-OCR/calamari#212]
Hi @ChWick / @andbue,

Is there any way to get Individual Character bounding coordinates for my post-processing?

Thanks.

**Comment by ChWick** [comment]
Have a look at the extended prediction data: https://github.com/Calamari-OCR/calamari/wiki/Extended-prediction-data

**Comment by andbue** [comment]
Does it still hold that "Calamari does whitespace post-processing on prediction.sentence, while it does not do the same on prediction.positions" [(ocrd_calamari)](https://github.com/OCR-D/ocrd_calamari/blob/da98713e73b998b6c5a10e341c9e6e0af59d84de/ocrd_calamari/recognize.py#L118)?

**Comment by ChWick** [comment]
@andbue Yes. Post-processing is only applied on the final sentence. Maybe we should adopt their code at least for a correct white-space post-processing.
